utils/erp.py

```diff
@@ -119,8 +120,14 @@ def push_fr_data(data, retired=False):
         "Ocp-Apim-Trace": "true",
         "Ocp-Apim-Subscription-Key": ERP_API_SUBSCRIPTION_KEY,
     }
-    # The response contains the GUID (res.text)
-    res = requests.post(ERP_API_ENDPOINT, json=payload, headers=headers)
+
+    # NOTE: Change this to an asynchronous call
+    try:
+        # The response contains the GUID (res.text)
+        res = requests.post(ERP_API_ENDPOINT, json=payload, headers=headers, timeout=ERP_API_MAX_REQ_TIMEOUT)
+        res.raise_for_status()
```

Comment:
This is not handled
https://3.python-requests.org/api/#requests.Response.raise_for_status